Sluggo/View Controllers/HomeTableViewController.swift

```diff
@@ -49,56 +49,74 @@ class HomeTableViewController: UITableViewController {
     func loadMember(completionHandler: (() -> Void)?) {
         let memberManager = MemberManager(identity: identity)
         memberManager.getMemberRecord(user: identity.authenticatedUser!, identity: identity) { result in
-            switch(result) {
-            case .success(let member):
-                DispatchQueue.main.sync {
-                    self.member = member
-                    completionHandler?()
-                }
-                break
-            case .failure(let error):
-                DispatchQueue.main.async {
-                    UIAlertController.createAndPresentError(vc: self, error: error, completion: nil)
-                }
-            }
+            self.processResult(result: result, onSuccess: { retrievedMember in
+                self.member = retrievedMember
+            }, after: completionHandler)
         }
     }
     
     func loadAssignedTickets(completionHandler: (() -> Void)?) {
-        // TODO
-        completionHandler?()
+        // Note: This will only grab the first page.
+        // This is an OK assumption, since we assume the total page size is > 3.
+        // Ideally, this could be extended with some other endpoints to get the total counts
+        // but this will suffice for now
+        let ticketsManager = TicketManager(identity)
```

Comment:
BTW: the paginated list object has a member, count, which is the total number in that collection